src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Added `IOpenTelemetryBuilder` interface to support authoring extensions which
+  can configure multiple OpenTelemetry signals (tracing, metrics, and/or logs).
+  ([#5265](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5265))
```

Comment:
I think this changelog entry was meant for OpenTelemetry.Api.Providerbuilders. But also, maybe no changelog is necessary yet? Is there really much an end user can do with `IOpenTelemetryBuilder` until more work is landed?

Reply:
> I think this changelog entry was meant for OpenTelemetry.Api.Providerbuilders.

Fixed thanks!

> But also, maybe no changelog is necessary yet? Is there really much an end user can do with IOpenTelemetryBuilder until more work is landed?

Yes! After this PR users can use `IOpenTelemetryBuilder` to authoring cross-cutting things. Like...

```csharp
public static IOpenTelemetryBuilder AddOtlpExporter(this IOpenTelemetryBuilder builder)
{
    builder.Services.ConfigureOpenTelemetryTracerProvider(builder => builder.AddOtlpExporter());
    builder.Services.ConfigureOpenTelemetryMeterProvider(builder => builder.AddOtlpExporter());
    builder.Services.ConfigureOpenTelemetryLoggerProvider(builder => builder.AddOtlpExporter());

    return builder;
}
```